### PR TITLE
fabtests: Correct help for fi_av_test

### PR DESCRIPTION
**Pull request by chien-intel**
Use argv[0] for name.  Remove extra "]" for -G.

Signed-off-by: Chien Tin Tung <chien.tin.tung@intel.com>

### CODE REVIEW

**Comment by j-xiong** [comment]
All the unit tests have the same usage() convention. Either keep it as it is or change them all.

File: fabtests/unit/getinfo_test.c [hunk 1]
```diff
@@ -1032,10 +1032,10 @@ int main(int argc, char **argv)
 			ft_parseinfo(op, optarg, hints, &opts);
 			break;
 		case 'h':
-			usage();
+			usage(argv[0]);
 			return EXIT_SUCCESS;
```
Comment (chien-intel) [inline fabtests/unit/getinfo_test.c:1036]: for some reason getinfo_test.c is different from others where -h returns EXIT_SUCCESS while others return EXIT_FAILURE

Comment (j-xiong) [reply fabtests/unit/getinfo_test.c:1036]: I think it is safe to make this consistent with others.

Comment (chien-intel) [reply fabtests/unit/getinfo_test.c]: done

**Review by j-xiong — APPROVED** [review]

**Pull request merged**